client/less/classic-modal.less

```diff
@@ -0,0 +1,45 @@
+.success {
```

Comment:
> These classes are still needed

If the modal works production then the css is already there and this a duplication. Please keep looking.